### PR TITLE
Generic ci

### PR DESCRIPTION
**Pull request by ChristopherMacGown**
Blah blah blah, CI interface. Better documentation. Better tests.

### CODE REVIEW

File: tests/daemon_unittest.py [hunk 1]
```diff
@@ -9,12 +10,17 @@ from tests import utils
 
 class DaemonTestCase(utils.TestHelper):
     def setUp(self):
+        self.config = Config(roundabout.config.DEFAULT)
+
+    def setUp(self):
```
Comment (larsbutler) [inline tests/daemon_unittest.py]: Why are there two setUp methods?

**Comment by larsbutler** [comment]
lgtm

**Comment by ChristopherMacGown** [comment]
Rejecting due to increased pylint score: 4 15

**Comment by ChristopherMacGown** [comment]
Will fix the new pylint changes in the fix_config_stupidity branch.

**Comment by ChristopherMacGown** [comment]
lgtm

**Comment by ChristopherMacGown** [comment]
Build failed, rejecting.

http://hudson.atomicpony.com/job/roundabout/124/

**Comment by ChristopherMacGown** [comment]
https://twitter.com/0x44/status/63278122455416832

I need better change management on my CI server's roundabout.cfg

**Comment by ChristopherMacGown** [comment]
lgtm

**Comment by ChristopherMacGown** [comment]
Build successful! Merged!

**Pull request closed without merging**